File: 1-two-sum/1-two-sum.py
class Solution(object):
    def twoSum(self, arr, target):
        final = []
        for i in range(len(arr) - 1):
            for j in range(i+1,len(arr)):
                if arr[i] + arr[j] == target:
                    final.append(i)
                    final.append(j)
        return final
                
                
                
                    
        
        
                    
                    
            
       
                    
                              
            

File: 1-two-sum/1-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> element = new HashMap();
        for(int i = 0; i < nums.length;i++){
            int solution = target - nums[i];
            if(element.containsKey(solution)){
                return new int[] {i,element.get(solution)};
            }
            
            element.put(nums[i],i);
            
        }
        
        
        return null;
        
    }
    
    
    
}




